src/main/scala/org/antipathy/scoozie/action/FsAction.scala

```diff
@@ -28,25 +28,26 @@ class FsAction(override val name: String,
 
   private val jobXmlProperty = buildStringOptionProperty(name, "jobXml", jobXmlOption)
 
-  private val namedActionsAnProps: Seq[(FileSystemAction, Map[String, String])] = steps.zipWithIndex.map {
+  private val namedActionsAnProps: Seq[ActionProperties[FileSystemAction]] = steps.zipWithIndex.map {
     case (Chmod(path, permissions, dirFiles), index) =>
+      import org.antipathy.scoozie.configuration.ActionProperties
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)